src/Cron_Event_Command.php

```diff
@@ -428,12 +430,22 @@ protected static function format_event( stdClass $event ) {
 	 *
 	 * @return array|WP_Error An array of event objects, or a WP_Error object if there are no events scheduled.
 	 */
-	protected static function get_cron_events() {
+	protected static function get_cron_events( $is_due_now = false ) {
+
+		if ( $is_due_now ) {
+			// wp_get_ready_cron_jobs since 5.1.0
+			if ( version_compare( get_bloginfo( 'version' ), '5.1', '>=' ) ) {
+				$crons = wp_get_ready_cron_jobs();
+			} else {
+				$crons = _get_cron_array();
+			}
+		} else {
+			$crons = _get_cron_array();
+		}
```

Comment:
A more concise way of writing this:

```php
// wp_get_ready_cron_jobs since 5.1.0
if ( $is_due_now && function_exists( 'wp_get_ready_cron_jobs' ) ) {
	$crons = wp_get_ready_cron_jobs();
} else {
	$crons = _get_cron_array();
}
```